otwcode/otwarchive
# AO3-6007 Show series counts on pseud dashboard page

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6007

## Purpose

Changes the heading for a pseud's series page to be "X Series by pseud (username) to match the same format as works or bookmarks pages.  Was previously just "pseud's series".

## Testing Instructions

1. Have a user with additional pseuds
2. View the series page for the user, the heading should be "X Series by username"
3. View the series page for a non-default pseud, the heading should be "X Series by pseud (username)"

The scenario outline in `user_dashboard.feature` from #3884 was leveraged to test this the same as other pseud pages.  A few other test case expectations were also modified to handle the counts in the heading now.

## References

See also AO3-6006 and #3884.  Same scenario outline is used for testing, since the expected outcome is similar.  There may be conflicts with that one depending on the merge order.


## Credit

tlee911 (he/him)
